abc.templates/drift-workflows/spec.yaml

```diff
@@ -54,7 +54,7 @@ steps:
         - to_replace: 'REPLACE_GUARDIAN_IGNORE_DIR_PATTERNS'
           with: '{{.guardian_ignore_dir_patterns}}'
         - to_replace: 'REPLACE_GUARDIAN_VERSION_TAG'
-          with: '{{._git_tag}}'
+          with: '{{ slice ._git_tag 1 }}'
```

Comment:
does this work?

Reply:
I was able to confirm this does work

Reply:
In the future, we can test this using goldentests to provide a test version tag